pkg/daemon/ceph/cleanup/radosnamespace.go

```diff
@@ -24,6 +24,12 @@ import (
 	cephclient "github.com/rook/rook/pkg/daemon/ceph/client"
 )
 
+const (
+
+	// ClientBlocklistDuration is the duration (in seconds) for which the client IP will be blocklisted
+	ClientBlocklistDuration = "1200"
```

Comment:
@travisn @sp98 Do you think it make sense to have option to override this using the annotation or the configmap so that we dont need any code changes if we get in to problem with time in some clusters?

Reply:
I don't think even making this timeout configurable will help. Setting a timeout that we think should be the max timeout required in most of the deletion scenarios should suffice.